packages/time-utilities/tests/lib/Cron.test.ts

```diff
@@ -0,0 +1,1365 @@
+import { Cron } from '../../src';
+
+describe('Cron', () => {
+	test('pre-defined @hourly', () => {
+		expect.assertions(6);
+
+		const specimine = new Cron('@hourly');
```

Comment:
Whats a `specimine`? you mean `specimen`? `entry`?

Reply:
`cron`?

Reply:
Apparently it's the singular of `specimin`

Reply:
`specimin` doesn't exist..

Reply:
specimen, whatever